In [1]:
import pandas as pd

# #features description

**categorical Attributes**

**name of countires and there are  24 countries**

Argentina, Australia, Austria, Brazil, Canada, Dominican, Filipino, Finland, France, Germany, Ireland, Italy, Mexico, Northern Ireland, Philippines, Portugal, Russia, Serbia, Spain, Switzerland, UK, USA, 

# sport

american Football, American Football / Baseball, Auto Racing, Auto Racing (Nascar), Auto racing, Baseball, Basketball, Boxing, F1 Motorsports, F1 racing, Golf, Hockey, Ice Hockey, MMA, NASCAR, NBA, NFL, Soccer, Tennis, auto racing, baseball, basketball, boxing, cycling, golf, ice hockey, motorcycle gp, soccer, tennis, 
​

In [2]:
df= pd.read_csv('forbes.csv')
df.head(50)

S.NO                     Name Nationality  Current Rank  \
0      1               Mike Tyson         USA             1   
1      2           Buster Douglas         USA             2   
2      3        Sugar Ray Leonard         USA             3   
3      4             Ayrton Senna      Brazil             4   
4      5              Alain Prost      France             5   
5      6            Jack Nicklaus         USA             6   
6      7              Greg Norman   Australia             7   
7      8           Michael Jordan         USA             8   
8      9            Arnold Palmer         USA             8   
9     10        Evander Holyfield         USA             8   
10    11        Evander Holyfield         USA             1   
11    12               Mike Tyson         USA             2   
12    13           Michael Jordan         USA             3   
13    14           George Foreman         USA             4   
14    15             Ayrton Senna      Brazil             5   
15    16              Alain Prost      France             6   
16    17  Donovan "Razor" Ruddock      Canada             7   
17    18            Arnold Palmer         USA             8   
18    19            Nigel Mansell          UK             9   
19    20        Sugar Ray Leonard         USA            10   
20    21           Michael Jordan         USA             1   
21    22        Evander Holyfield         USA             2   
22    23             Ayrton Senna      Brazil             3   
23    24            Nigel Mansell          UK             4   
24    25            Arnold Palmer         USA             5   
25    26             Andre Agassi         USA             6   
26    27              Joe Montana         USA             7   
27    28        Sugar Ray Leonard         USA             8   
28    29              Jim Courier         USA             9   
29    30             Monica Seles         USA            10   
30    31           Michael Jordan         USA             1   
31    32             Riddick Bowe         USA             2   
32    33             Ayrton Senna      Brazil             3   
33    34              Alain Prost      France             4   
34    35           George Foreman         USA             5   
35    36         Shaquille O'Neal         USA             6   
36    37             Lennox Lewis         USA             7   
37    38            Cecil Fielder         USA             8   
38    39              Jim Courier         USA             9   
39    40              Joe Montana         USA            10   
40    41           Michael Jordan         USA             1   
41    42              Shaq O'Neal         USA             2   
42    43            Jack Nicklaus         USA             3   
43    44            Arnold Palmer         USA             4   
44    45           Gerhard Berger     Austria             5   
45    46            Wayne Gretzky      Canada             5   
46    47           Michael Moorer         USA             7   
47    48        Evander Holyfield         USA             8   
48    49             Andre Agassi         USA             9   
49    50            Nigel Mansell          UK            10   

   Previous Year Rank        Sport  Year  earnings ($ million)  
0                 NaN       boxing  1990                  28.6  
1                 NaN       boxing  1990                  26.0  
2                 NaN       boxing  1990                  13.0  
3                 NaN  auto racing  1990                  10.0  
4                 NaN  auto racing  1990                   9.0  
5                 NaN         golf  1990                   8.6  
6                 NaN         golf  1990                   8.5  
7                 NaN   basketball  1990                   8.1  
8                 NaN         golf  1990                   8.1  
9                 NaN       boxing  1990                   8.1  
10                  8       boxing  1991                  60.5  
11        

In [3]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 301 entries, 0 to 300
Data columns (total 8 columns):
 #   Column                Non-Null Count  Dtype  
---  ------                --------------  -----  
 0   S.NO                  301 non-null    int64  
 1   Name                  301 non-null    object 
 2   Nationality           301 non-null    object 
 3   Current Rank          301 non-null    int64  
 4   Previous Year Rank    277 non-null    object 
 5   Sport                 301 non-null    object 
 6   Year                  301 non-null    int64  
 7   earnings ($ million)  301 non-null    float64
dtypes: float64(1), int64(3), object(4)
memory usage: 18.9+ KB


In [4]:
df.shape

(301, 8)

In [5]:
df_num=df.copy()

In [6]:
df.describe()

S.NO  Current Rank         Year  earnings ($ million)
count  301.000000    301.000000   301.000000            301.000000
mean   151.000000      5.448505  2005.122924             45.516279
std     87.035433      2.850995     9.063563             33.525337
min      1.000000      1.000000  1990.000000              8.100000
25%     76.000000      3.000000  1997.000000             24.000000
50%    151.000000      5.000000  2005.000000             39.000000
75%    226.000000      8.000000  2013.000000             59.400000
max    301.000000     10.000000  2020.000000            300.000000

In [7]:
df.describe(include=['O'])

Name Nationality Previous Year Rank       Sport
count           301         301                277         301
unique           82          22                 36          29
top     Tiger Woods         USA                >10  Basketball
freq             19         206                 36          54

In [8]:
df.isnull().sum()

S.NO                     0
Name                     0
Nationality              0
Current Rank             0
Previous Year Rank      24
Sport                    0
Year                     0
earnings ($ million)     0
dtype: int64

It seems that the null values exist because these athletes weren't previously ranked so we do not have to worry about the null values

# datA CLEANING

In [9]:
import pandas as pd 



#Print Categorical attributes
print("Categorical Attributes:")
for cols in df.columns:
    if df[cols].dtype == 'object':
        print(cols)

Categorical Attributes:
Name
Nationality
Previous Year Rank
Sport


In [10]:
df.drop(['Previous Year Rank'],axis=1,inplace=True)

In [11]:
year_n=df.sort_values('Year',ascending=True).head(15)
    
year_n

S.NO                     Name Nationality  Current Rank        Sport  \
0      1               Mike Tyson         USA             1       boxing   
1      2           Buster Douglas         USA             2       boxing   
2      3        Sugar Ray Leonard         USA             3       boxing   
3      4             Ayrton Senna      Brazil             4  auto racing   
4      5              Alain Prost      France             5  auto racing   
5      6            Jack Nicklaus         USA             6         golf   
6      7              Greg Norman   Australia             7         golf   
7      8           Michael Jordan         USA             8   basketball   
8      9            Arnold Palmer         USA             8         golf   
9     10        Evander Holyfield         USA             8       boxing   
19    20        Sugar Ray Leonard         USA            10         golf   
18    19            Nigel Mansell          UK             9  auto racing   
17    18            Arnold Palmer         USA             8         golf   
15    16              Alain Prost      France             6  auto racing   
16    17  Donovan "Razor" Ruddock      Canada             7       boxing   

    Year  earnings ($ million)  
0   1990                  28.6  
1   1990                  26.0  
2   1990                  13.0  
3   1990                  10.0  
4   1990                   9.0  
5   1990                   8.6  
6   1990                   8.5  
7   1990                   8.1  
8   1990                   8.1  
9   1990                   8.1  
19  1991                   8.5  
18  1991                   9.0  
17  1991                   9.3  
15  1991                  11.0  
16  1991                  10.2

In [12]:
dfname = df.groupby(['Sport'])
dfname1 = dfname['Sport'].count().reset_index(name='Count').sort_values('Count', ascending=False).head(20)
dfname1


Sport  Count
6          Basketball     54
7              Boxing     29
21         basketball     27
10               Golf     24
17             Soccer     22
24               golf     20
18             Tennis     18
0   American Football     17
22             boxing     17
27             soccer     11
2         Auto Racing     10
9           F1 racing      8
19        auto racing      7
28             tennis      5
8      F1 Motorsports      5
26      motorcycle gp      4
16                NFL      3
20           baseball      3
5            Baseball      3
14             NASCAR      3

In [13]:
y= df.groupby('Year')
y

In [14]:
df.columns

Index(['S.NO', 'Name', 'Nationality', 'Current Rank', 'Sport', 'Year',
       'earnings ($ million)'],
      dtype='object')

In [15]:
df.isnull()

S.NO   Name  Nationality  Current Rank  Sport   Year  \
0    False  False        False         False  False  False   
1    False  False        False         False  False  False   
2    False  False        False         False  False  False   
3    False  False        False         False  False  False   
4    False  False        False         False  False  False   
..     ...    ...          ...           ...    ...    ...   
296  False  False        False         False  False  False   
297  False  False        False         False  False  False   
298  False  False        False         False  False  False   
299  False  False        False         False  False  False   
300  False  False        False         False  False  False   

     earnings ($ million)  
0                   False  
1                   False  
2                   False  
3                   False  
4                   False  
..                    ...  
296                 False  
297                 False  
298                 False  
299                 False  
300                 False  

[301 rows x 7 columns]

In [16]:
#Checking for duplicate data
df.duplicated().sum()

0

In [17]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

import plotly.figure_factory as ff
import plotly.graph_objects as go
import plotly.express as px
from plotly.subplots import make_subplots
from matplotlib import style
style.use('fivethirtyeight') # fivethirtyeight

import plotly.io as pio
pio.renderers
pio.renderers.default = "notebook_connected"

In [18]:
dfname = df.groupby(['Name'])
dfname1 = dfname['Name'].count().reset_index(name='Count').sort_values('Count', ascending=False).head(10)
dfname1

fig = px.bar(dfname1, x = 'Count', y = 'Name', title= 'Atheltes who have appeared the most on Forbes highest paid atheletes list', labels={"Name": "Atheltes", "Count": "Number of appearances"})
fig.update_layout(yaxis_categoryorder = 'total ascending')
fig.show()

Both Tiger Woods and Michael Jordan have appeared on the list 19 times, which is the highest number of times an athelete has appeared on the list.

In [19]:
money = df.groupby('Name')[['earnings ($ million)']].sum().reset_index().sort_values('earnings ($ million)', ascending=False).head(20)

fig = px.bar(money, x = 'earnings ($ million)', y = 'Name', title= 'Total earnings for Atheletes on the Forbes list between 1990-2020', labels={"Name": "Athletes", "earnings ($ million)": "Total earnings (in US$ million)"})
fig.update_layout(yaxis_categoryorder = 'total ascending')
fig.show()

tiger Woods is on top again, he has earned the most and also appeared the most in forbes list. While Michael Jordan has appeared as many times as Tiger Woods, he has earned about $500 Million less than Tiger Woods in his career in forbes list.


# Who are the athletes who have placed rank in Forbes list and how many times?

In [20]:
rank = df[df['Current Rank']== 1]

fig = px.bar(rank, x = 'Current Rank', y = 'Name', title='Athletes who placed rank 1 in Forbes list', labels={"Name": "Athletes", "Current Rank": "Times atheletes placed rank 1"})
fig.update_layout(yaxis_categoryorder = 'total ascending')
fig.show()


Tiger Woods is on top again,

# Which countries have the high earning athletes and how much have they earned?

In [21]:
countries = dfname = df.groupby(['Nationality'])
nationality = dfname['Nationality'].count().reset_index(name='Count').sort_values('Count', ascending=False).head(10)

fig = px.bar(nationality, x = 'Count', y = 'Nationality', title= 'Amount of times atheltes from these countries have appeared on the list')
fig.update_layout(yaxis_categoryorder = 'total ascending')
fig.show()

most athletes who have appered on the list are from USA

In [22]:
#Fixing the names of the sport to group them together
df['Sport'] = df['Sport'].str.capitalize()


sport = df.groupby('Sport')[['earnings ($ million)']].sum().reset_index().sort_values('earnings ($ million)', ascending=False).head(15)

fig = px.bar(sport, x = 'earnings ($ million)', y = 'Sport', title= 'Total earnings for Atheletes on the Forbes list between 1990-2020', labels={"Name": "Athletes", "earnings ($ million)": "Total earnings (in US$ million)"})
fig.update_layout(yaxis_categoryorder = 'total ascending')
fig.show()

the highest earning game was basketball.

In [23]:
top_earnings=df.sort_values(by = "earnings ($ million)",ascending=False)
top_earnings.head()
# $ 300.0 millions is the top paid amount in the field of boxing and ppaid to the player Floyd Mayweather.

S.NO              Name  Nationality  Current Rank   Sport  Year  \
241   242  Floyd Mayweather          USA             1  Boxing  2015   
271   272  Floyd Mayweather          USA             1  Boxing  2018   
242   243    Manny Pacquiao  Philippines             2  Boxing  2015   
281   282      Lionel Messi    Argentina             1  Soccer  2019   
171   172       Tiger Woods          USA             1    Golf  2008   

     earnings ($ million)  
241                 300.0  
271                 285.0  
242                 160.0  
281                 127.0  
171                 115.0

In [24]:
top_earnings.tail()
# $ 8.1 millions is the least amount paid to Michael Jordan in the field of basketball

S.NO               Name Nationality  Current Rank       Sport  Year  \
19    20  Sugar Ray Leonard         USA            10        Golf  1991   
29    30       Monica Seles         USA            10      Tennis  1992   
9     10  Evander Holyfield         USA             8      Boxing  1990   
8      9      Arnold Palmer         USA             8        Golf  1990   
7      8     Michael Jordan         USA             8  Basketball  1990   

    earnings ($ million)  
19                   8.5  
29                   8.5  
9                    8.1  
8                    8.1  
7                    8.1

In [25]:
df_golf= df[df["Sport"]=="Golf"]
df_golf.sort_values(by="earnings ($ million)",ascending=False
                    
        
                   
                   
                   )
# in the field of golf sport $ 115.0 million is the top amount been paid to the players and $ 8.1 million is the least amount been paid

S.NO               Name       Nationality  Current Rank Sport  Year  \
171   172        Tiger Woods               USA             1  Golf  2008   
181   182        Tiger Woods               USA             1  Golf  2009   
191   192        Tiger Woods               USA             1  Golf  2010   
161   162        Tiger Woods               USA             1  Golf  2007   
151   152        Tiger Woods               USA             1  Golf  2006   
141   142        Tiger Woods               USA             1  Golf  2005   
131   132        Tiger Woods               USA             1  Golf  2004   
221   222        Tiger Woods               USA             1  Golf  2013   
121   122        Tiger Woods               USA             1  Golf  2003   
201   202        Tiger Woods               USA             1  Golf  2011   
110   111        Tiger Woods               USA             1  Golf  2002   
298   299        Tiger Woods               USA             8  Golf  2020   
236   237        Tiger Woods               USA             6  Golf  2014   
213   214        Tiger Woods               USA             3  Golf  2012   
238   239     Phil Mickelson               USA             8  Golf  2014   
101   102        Tiger Woods               USA             2  Golf  2000   
258   259     Phil Mickelson               USA             8  Golf  2016   
259   260      Jordan Spieth               USA             9  Golf  2016   
248   249     Phil Mickelson               USA             8  Golf  2015   
249   250        Tiger Woods               USA             9  Golf  2015   
267   268       Rory McIlroy  Northern Ireland             6  Golf  2017   
227   228     Phil Mickelson               USA             7  Golf  2013   
217   218     Phil Mickelson               USA             7  Golf  2012   
154   155     Phil Mickelson               USA             4  Golf  2006   
91     92        Tiger Woods               USA             2  Golf  1999   
205   206     Phil Mickelson               USA             5  Golf  2011   
194   195     Phil Mickelson               USA             4  Golf  2010   
174   175     Phil Mickelson               USA             3  Golf  2008   
163   164     Phil Mickelson               USA             3  Golf  2007   
187   188     Phil Mickelson               USA             6  Golf  2009   
83     84        Tiger Woods               USA             4  Golf  1998   
75     76        Tiger Woods               USA             6  Golf  1997   
88     89      Arnold Palmer               USA             9  Golf  1998   
57     58  Sugar Ray Leonard               USA             8  Golf  1995   
67     68      Arnold Palmer               USA             8  Golf  1996   
42     43      Jack Nicklaus               USA             3  Golf  1994   
43     44      Arnold Palmer               USA             4  Golf  1994   
24     25      Arnold Palmer               USA             5  Golf  1992   
17     18      Arnold Palmer               USA             8  Golf  1991   
27     28  Sugar Ray Leonard               USA             8  Golf  1992   
5       6      Jack Nicklaus               USA             6  Golf  1990   
6       7        Greg Norman         Australia             7  Golf  1990   
19     20  Sugar Ray Leonard               USA            10  Golf  1991   
8       9      Arnold Palmer               USA             8  Golf  1990   

     earnings ($ million)  
171                 115.0  
181                 110.0  
191                 105.0  
161                 100.0  
151                  90.0  
141                  87.0  
131                  80.3  
221                  78.1  
121                  78.0  
201                  75.0  
110                  69.0  
298                  62.3  
236                  61.2  
213                  59.4  
238                  53.2  
101                  53.0  
258                  52.9  
259                  52.8  
248                  50.8  
249                  50.

In [26]:
rich_boxing=df[df["Sport"]=="Boxing"]
rich_boxing.sort_values(by="earnings ($ million)",ascending=False)
# in the field of boxing sport $ 300.0 million is the top amount been paid to the players and $ 8.1 million is the least amount been paid

S.NO                     Name  Nationality  Current Rank   Sport  Year  \
241   242         Floyd Mayweather          USA             1  Boxing  2015   
271   272         Floyd Mayweather          USA             1  Boxing  2018   
242   243           Manny Pacquiao  Philippines             2  Boxing  2015   
231   232         Floyd Mayweather          USA             1  Boxing  2014   
284   285           Canelo Alvarez       Mexico             4  Boxing  2019   
211   212         Floyd Mayweather          USA             1  Boxing  2012   
60     61               Mike Tyson          USA             1  Boxing  1996   
192   193         Floyd Mayweather          USA             2  Boxing  2010   
212   213           Manny Pacquiao     Filipino             2  Boxing  2012   
10     11        Evander Holyfield          USA             1  Boxing  1991   
153   154             Muhammad Ali          USA             3  Boxing  2006   
71     72        Evander Holyfield          USA             2  Boxing  1997   
102   103               Mike Tyson          USA             3  Boxing  2000   
92     93         Oscar De La Hoya          USA             3  Boxing  1999   
162   163         Oscar De La Hoya          USA             2  Boxing  2007   
198   199           Manny Pacquiao  Philippines             8  Boxing  2010   
51     52               Mike Tyson          USA             2  Boxing  1995   
188   189           Manny Pacquiao  Philippines             6  Boxing  2009   
72     73         Oscar De La Hoya          USA             3  Boxing  1997   
143   144         Oscar De La Hoya          USA             3  Boxing  2005   
94     95        Evander Holyfield          USA             5  Boxing  1999   
95     96               Mike Tyson          USA             6  Boxing  1999   
11     12               Mike Tyson          USA             2  Boxing  1991   
125   126         Oscar De La Hoya          USA             5  Boxing  2003   
97     98             Lennox Lewis          USA             8  Boxing  1999   
0       1               Mike Tyson          USA             1  Boxing  1990   
113   114             Lennox Lewis          USA             4  Boxing  2002   
21     22        Evander Holyfield          USA             2  Boxing  1992   
74     75               Mike Tyson          USA             5  Boxing  1997   
1       2           Buster Douglas          USA             2  Boxing  1990   
31     32             Riddick Bowe          USA             2  Boxing  1993   
116   117               Mike Tyson          USA             7  Boxing  2002   
108   109         Oscar De La Hoya          USA             9  Boxing  2000   
107   108             Lennox Lewis          USA             8  Boxing  2000   
53     54             Riddick Bowe          USA             4  Boxing  1995   
86     87         Oscar De La Hoya          USA             7  Boxing  1998   
55     56           George Foreman          USA             6  Boxing  1995   
34     35           George Foreman          USA             5  Boxing  1993   
65     66        Evander Holyfield          USA             6  Boxing  1996   
36     37             Lennox Lewis          USA             7  Boxing  1993   
13     14           George Foreman          USA             4  Boxing  1991   
2       3        Sugar Ray Leonard          USA             3  Boxing  1990   
46     47           Michael Moorer          USA             7  Boxing  1994   
47     48        Evander Holyfield          USA             8  Boxing  1994   
16     17  Donovan "Razor" Ruddock       Canada             7  Boxing  1991   
9      10        Evander Holyfield          USA             8  Boxing  1990   

     earnings ($ million)  
241                 300.0  
271                 285.0  
242                 160.0  
231                 105.0  
284                  94.0  
211                  85.0  
60                   75.0  
192                  65.0  
212                  62.0  
10         

In [27]:
rich_basketball=df[df["Sport"]=="Basketball"]
rich_basketball.sort_values(by="earnings ($ million)",ascending=False)
# in the field of basketball sport $ 89.0 million is the top amount been paid to the players and $ 8.1 million is the least amount been paid

S.NO            Name Nationality  Current Rank       Sport  Year  \
288   289    LeBron James         USA             8  Basketball  2019   
295   296    LeBron James         USA             5  Basketball  2020   
262   263    LeBron James         USA             2  Basketball  2017   
276   277    LeBron James         USA             6  Basketball  2018   
289   290   Stephen Curry         USA             9  Basketball  2019   
..    ...             ...         ...           ...         ...   ...   
41     42     Shaq O'Neal         USA             2  Basketball  1994   
12     13  Michael Jordan         USA             3  Basketball  1991   
68     69   Dennis Rodman         USA             9  Basketball  1996   
69     70   Patrick Ewing         USA             9  Basketball  1996   
7       8  Michael Jordan         USA             8  Basketball  1990   

     earnings ($ million)  
288                  89.0  
295                  88.2  
262                  86.2  
276                  85.5  
289                  79.8  
..                    ...  
41                   16.7  
12                   16.0  
68                   12.9  
69                   12.9  
7                     8.1  

[81 rows x 7 columns]

In [28]:
rich_soccer=df[df["Sport"]=="Soccer"]
rich_soccer.sort_values(by="earnings ($ million)",ascending=False)
# in the field of soccer sport $ 127.0 million is the top amount been paid to the players and $ 28 million is the least amount been paid

S.NO               Name Nationality  Current Rank   Sport  Year  \
281   282       Lionel Messi   Argentina             1  Soccer  2019   
272   273       Lionel Messi   Argentina             2  Soccer  2018   
282   283  Cristiano Ronaldo    Portugal             2  Soccer  2019   
273   274  Cristiano Ronaldo    Portugal             3  Soccer  2018   
292   293  Cristiano Ronaldo    Portugal             2  Soccer  2020   
283   284             Neymar      Brazil             3  Soccer  2019   
293   294       Lionel Messi   Argentina             3  Soccer  2020   
294   295             Neymar      Brazil             4  Soccer  2020   
261   262  Cristiano Ronaldo    Portugal             1  Soccer  2017   
275   276             Neymar      Brazil             5  Soccer  2018   
251   252  Cristiano Ronaldo    Portugal             1  Soccer  2016   
252   253       Lionel Messi   Argentina             2  Soccer  2016   
263   264       Lionel Messi   Argentina             3  Soccer  2017   
232   233  Cristiano Ronaldo    Portugal             2  Soccer  2014   
243   244  Cristiano Ronaldo    Portugal             3  Soccer  2015   
244   245       Lionel Messi   Argentina             4  Soccer  2015   
234   235       Lionel Messi   Argentina             4  Soccer  2014   
172   173      David Beckham          UK             2  Soccer  2008   
228   229      David Beckham          UK             8  Soccer  2013   
218   219      David Beckham          UK             8  Soccer  2012   
229   230  Cristiano Ronaldo    Portugal             9  Soccer  2013   
195   196      David Beckham          UK             5  Soccer  2010   
219   220  Cristiano Ronaldo    Portugal             9  Soccer  2012   
185   186      David Beckham          UK             5  Soccer  2009   
230   231       Lionel Messi   Argentina            10  Soccer  2013   
206   207      David Beckham          UK             6  Soccer  2011   
207   208  Cristiano Ronaldo    Portugal             7  Soccer  2011   
178   179         Ronaldinho      Brazil             8  Soccer  2008   
166   167      David Beckham          UK             6  Soccer  2007   
147   148      David Beckham          UK             7  Soccer  2005   
210   211       Lionel Messi   Argentina            10  Soccer  2011   
169   170         Ronaldinho      Brazil             9  Soccer  2007   
138   139      David Beckham          UK             8  Soccer  2004   

     earnings ($ million)  
281                 127.0  
272                 111.0  
282                 109.0  
273                 108.0  
292                 105.0  
283                 105.0  
293                 104.0  
294                  95.5  
261                  93.0  
275                  90.0  
251                  88.0  
252                  81.4  
263                  80.0  
232                  80.0  
243                  79.6  
244                  73.8  
234                  64.7  
172                  50.0  
228                  47.2  
218                  46.0  
229                  44.0  
195                  43.7  
219                  42.5  
185                  42.0  
230                  41.3  
206                  40.0  
207                  38.0  
178                  37.0  
166                  33.0  
147                  32.5  
210                  32.3  
169                  31.0  
138                  28.0

conclusions:
out of 301 players most of them are usa players-->206 players are from usa.

Basketball is the most popular sport among the dataset


2002 is the most repeated year.

there is no row where current rank is equal to previous year rank.

tiger woods been the top earnings most of the time.

  300.0 millions is the top paid amount in the field of boxing and ppaid to the player Floyd Mayweather.
  8.1 millions is the least amount paid to Michael Jordan in the field of basketball.
 
 in the field of golf sport  115.0 million is the top amount been paid to the players and 8.1 million is the least amount been paid.
 
 in the field of boxing sport 300.0 million is the top amount been paid to the players and 12.0 million is the least amount been paid.
 
 in the field of basketball sport  89.0 million is the top amount been paid to the players and $12.9 million is the least amount been paid.
 
in the field of soccer sport $127.0 million is the top amount been paid to the players and 41.3 million is the least amount been paid.

 tigher woods was the player who became no more than 10 time in rank 1 and highest appernece in frobes list.

